config/settings.yml

```diff
@@ -74,6 +74,13 @@ dmc:
   url: https://internal-dsva-vagov-dev-fwdproxy-1893365470.us-gov-west-1.elb.amazonaws.com:4465/api/v1/digital-services/
   debts_endpoint: debt-letter/get
 
+mcp:
```

Comment:
Will there be entries for these keys in devops repo, if so, could you provide the link in the PR desc?

Reply:
Yes, the entries will be made in the DevOps repo once the Liberty team provides us the configuration data for the VBS service. For now, everything will be hidden behind a feature flip.

Reply:
I've updated the PR description to reflect my above comment.